Patch-Source: https://github.com/l4l/yofi/pull/131
--
From b1ca3181afcd7aa39186b3e358890cd03dd52082 Mon Sep 17 00:00:00 2001
From: Jakub Jirutka <jakub@jirutka.cz>
Date: Wed, 19 Oct 2022 21:58:36 +0200
Subject: [PATCH] Bump png to 0.17.6

The new version dropped dependency on a quite big encoding crate.

diff --git a/Cargo.toml b/Cargo.toml
index d9a1e64..7227979 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -33 +33 @@ serde = { version = "1.0.131", features = ["derive"] }
-png = { version = "0.17.2", default-features = false }
+png = { version = "0.17.6", default-features = false }
